### ISSUE TITLE
[feat] Bulk get/set config parameters

### ISSUE THREAD

**Issue by robertsLando** [zwave-js/zwavejs2mqtt#1366]
Ref: https://github.com/zwave-js/node-zwave-js/pull/2875

**Comment by johanschelin** [comment]
Yes this would be really nice, for example as I have 50+ units where I would like to change the temp/watt change reports to be less frequent to easen the network traffic.

**Comment by robertsLando** [comment]
I still dunno what could be the right way to implement this on the UI. BTW you already can do this by using a driver function :)

**Comment by LordMike** [comment]
I'm not sure if this feature is for bulk setting _different_ parameters, or to be able to set one parameter that may be split into multiple sub-values.... assuming the latter, I imagine that for the MQTT transports, at least, you could re-support the configuration parameter itself, rather than the individual bits and bytes the config files describe.

Example:

```
Today:
zwavejs2mqtt/NODE/112/0/26/65280/set
zwavejs2mqtt/NODE/112/0/26/16711680/set
zwavejs2mqtt/NODE/112/0/26/4278190080/set

New:
zwavejs2mqtt/NODE/112/0/26/set <-- sets the "raw" byte value.
```

**Comment by robertsLando** [comment]
> I'm not sure if this feature is for bulk setting different parameters, or to be able to set one parameter that may be split into multiple sub-values

Both
